test/commands/positive.test.ts

```diff
@@ -421,7 +421,7 @@ describe('Positive', () => {
 
       await Positive.run(['https://url-does-not-matter.com']);
 
-      expect(mockPrompt).toHaveBeenCalledTimes(1);
+      expect(mockPrompt).toHaveBeenCalledTimes(2);
```

Comment:
Please verify that this is ok.

Reply:
The other mockPrompt further down the code has a similar 2x call.